app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesViewHolder.kt

```diff
@@ -52,9 +52,15 @@ class PocketStoriesViewHolder(
     override fun Content() {
         val horizontalPadding = dimensionResource(R.dimen.home_item_horizontal_margin)
 
+        val homeScreenReady = components.appStore
+            .observeAsComposableState { state -> state.firstFrameDrawn }.value ?: false
+
         val stories = components.appStore
             .observeAsComposableState { state -> state.pocketStories }.value
 
+        // Perceived performance can be improved by waiting to draw pocket content until after the
+        // first frame is drawn. See https://github.com/mozilla-mobile/fenix/issues/25545 for details.
```

Comment:
non-blocking: if you take my suggestion above to elaborate, I'd prefer if these two comments point to a central elaboration comment rather than these two comments be duplicated and elaborate